app/views/admin/document_collection_group_memberships/index.html.erb

```diff
@@ -26,17 +26,15 @@
             text: "Documents",
             margin_bottom: 6,
           } %>
-
-          <div class="app-view-document-collection-group-memberships-index__heading-links">
-            <p class="govuk-body govuk-!-text-align-right">
+          <ul class="govuk-list app-view-document-collection-group-memberships-index__heading-links">
```

Comment:
If we use the approach above we can lose the `app-view-document-collection-group-memberships-index__heading-links` class here.